controllers/reconcilers/kserve_istio_smmr_reconciler.go

```diff
@@ -72,7 +68,7 @@ func (r *KserveIstioSMMRReconciler) Reconcile(ctx context.Context, log logr.Logg
 }
 
 func (r *KserveIstioSMMRReconciler) createDesiredResource(ctx context.Context, log logr.Logger, isvc *kservev1beta1.InferenceService) (*v1.ServiceMeshMemberRoll, error) {
-	desiredSMMR, err := r.smmrHandler.FetchSMMR(ctx, log, types.NamespacedName{Name: serviceMeshMemberRollName, Namespace: constants.IstioNamespace})
+	desiredSMMR, err := r.smmrHandler.FetchSMMR(ctx, log, types.NamespacedName{Name: constants.ServiceMeshMemberRollName, Namespace: constants.IstioNamespace})
```

Comment:
A small nitpick on the choice of name: `ServiceMeshMemberRollName` shouldn't it be `ServiceMeshMemberRoleName`? What does Roll stand for in this context ?

Reply:
`Roll` is the right name :)
See https://docs.openshift.com/container-platform/4.15/service_mesh/v2x/ossm-create-mesh.html#ossm-about-adding-projects-using-smmr_ossm-create-mesh.